lte/cloud/go/services/lte/obsidian/handlers/handlers.go

```diff
@@ -300,21 +304,41 @@ func listEnodebs(c echo.Context) error {
 	if nerr != nil {
 		return nerr
 	}
+	var pageSize uint64 = 0
+	var err error
```

Comment:
No need to explicitly define this here

Reply:
this was a pattern used in the reference pr
https://github.com/magma/magma/pull/5381/files#diff-ccdd661b3d08b3d70276e71041e28714a15c9e8c1e10cf260b162a84af292b43R211-R219

im fine with refactoring this but i think we should conform to a pattern throughout the apis when calculating page sizes.
that said will pull this out into its own helper to make that easier going forward